src/lib/rpc/client.ts

```diff
@@ -68,6 +69,11 @@ export const createUserServiceClient = (token: string) => {
   return createClient(UserService, transport);
 };
 
+export function createProjectListClient(tenant: string, token: string) {
+  const transport = createTransport(apiBaseUrl, tenant, token);
```

Comment:
Use `cloudApiBaseUrl ` while creating project service client. Also it should be `createProjctServiceClient`. All operations within project service is available in this client. It is not specific to `List` operation